modules/37_industry/subsectors/datainput.gms

```diff
@@ -49,9 +57,144 @@ if (cm_IndCCSscen eq 1,
 *** assume 50 year lifetime for industry energy efficiency capital
 pm_delta_kap(regi,ppfKap_industry_dyn37) = -log(1 / 4) / 50;
 
-p37_energy_limit("ue_cement","en_cement")                     = 100;
-p37_energy_limit("ue_steel_primary","en_steel_primary")       = 100;
-p37_energy_limit("ue_steel_secondary","feel_steel_secondary") = 100;
+*** FIXME: this is temporary data, insert meaningful figures!
```

Comment:
Is this something you would like to change before the merge or not?

Reply:
I will work that out during "validation" of the realisation.